src/bin/sleep.rs

```diff
@@ -3,19 +3,86 @@
 extern crate coreutils;
 
 use std::env;
+use std::io::{self, Write};
 use std::thread;
-use std::io;
+use std::time::Duration;
 
-use coreutils::extra::{OptionalExt, fail};
 
-#[allow(deprecated)]
+use coreutils::extra::{OptionalExt};
+
+static MAN_PAGE: &'static str = r#"NAME
+    sleep - delay for a specified amount of time
+
+SYNOPSIS
+    sleep NUMBER[SUFFIX]...
```

Comment:
Okay, so there are a number things that differ:

1. SYNOPSIS is on more than one line, yet could be folded, since the latter is simply a special case of the former.
2. Each command alias are seperated by `, ` instead of line breaks.
3. Example do not have their own section.